src/DbSQLiteAdapter/Connection.cpp

```diff
@@ -4,20 +4,19 @@
 
 #include <DbAdapterInterface/IConnectionConfiguration.h>
 
+#include <excpt.h>
 #include <sqleet/sqleet.h>
 
-namespace systelab { namespace db { namespace sqlite {
-
-	Connection::Connection()
-	{
-	}
-
+namespace systelab::db::sqlite 
+{
+	Connection::Connection() = default;
```

Comment:
I don't see the reason to have ctor and dtor declared in the cpp, there is no incomplete type in this class. It would be better to remove both definitions.